<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutomationCreatePage</class>
 <widget class="QWidget" name="AutomationCreatePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
	color: #1A1A1A;
	font-family: Open Sans;
	font-weight: 600;
	font-size: 16px;
	line-height: 24px;
	letter-spacing: 0%;
}

QLineEdit {
	color: #1A1A1A;
	background: white;
	font-family: Open Sans;
	font-weight: 400;
	font-size: 16px;
	line-height: 24px;
	letter-spacing: 0%;
	border-radius: 8px;
	border: 1px solid #CCCCCC;
	padding-left: 12px;
	padding-right: 12px;
}

QTextEdit {
	color: #1A1A1A;
	background: white;
	font-family: Open Sans;
	font-weight: 400;
	font-size: 16px;
	line-height: 24px;
	letter-spacing: 0%;
	border-radius: 8px;
	border: 1px solid #CCCCCC;
	padding-left: 12px;
	padding-right: 12px;
}

QComboBox {
    background-color: white;
    border: 1px solid #ccc;
    border-radius: 8px;
    padding: 4px;
    font-size: 16px;
	line-height: 24px;
	letter-spacing: 0%;
    color: #333;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 24px;
    border: none;
}

QComboBox::down-arrow {
    image: url(:/images/chevron-down.svg);
    width: 16px;
    height: 16px;
}

QComboBox::down-arrow:on {
    image: url(:/images/chevron-up.svg);
}

QComboBox QAbstractItemView {
    background-color: white;
    margin-top: 0px;
    border: 1px solid #ccc;
    border-radius: 8px;
	outline: none;
}

/* Each dropdown item */
QComboBox QAbstractItemView::item {
	color: #1A1A1A;
    padding: 6px 12px;
    border-radius: 4px;
}

/* Hovered item */
QComboBox QAbstractItemView::item:hover {
    background-color: #e6f2ff;
}

/* Selected item */
QComboBox QAbstractItemView::item:selected {
    background-color: #d0e8ff;
    color: #1a1a1a;
}</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>18</number>
   </property>
   <property name="leftMargin">
    <number>18</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>18</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="headerLayout">
     <property name="spacing">
      <number>16</number>
     </property>
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>34</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>251</width>
         <height>34</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;
border: none;
font-family: Open Sans;
font-size: 24px;
line-height: 34px;
letter-spacing: 0%;
font-weight: 600;</string>
       </property>
       <property name="text">
        <string>Create Automation</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../Resources/Resources.qrc">
         <normaloff>:/images/nav_back.svg</normaloff>:/images/nav_back.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>155</width>
         <height>42</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>155</width>
         <height>42</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background: white;
	border: 2px solid #1A1A1A;
	border-radius: 8px;
	font-family: Open Sans;
	font-weight: 500;
	font-size: 18px;
	color: #1A1A1A;
}

QPushButton:hover {
    background-color: #D6ECFF;
}

QPushButton:pressed {
    background-color: #B7DCFF;
    padding-top: 2px;
	padding-left: 2px;
}
</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="minimumSize">
        <size>
         <width>155</width>
         <height>42</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>155</width>
         <height>42</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background: #FDB515;
	border: none;
	border-radius: 8px;
	font-family: Open-Sans;
	font-weight: 500;
	font-size: 18px;
	color: white;
}

QPushButton:hover {
    background-color: #E6A512;
}

QPushButton:pressed {
    background-color: #CC8F0F;
    padding-top: 2px;
	padding-left: 2px;
}</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>21</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="automationNamePartLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="automationNameLabel">
         <property name="text">
          <string>Automation Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="titleEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Enter automation name here</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="aiModelsPartLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="aiModelLabel">
         <property name="text">
          <string>AI Models</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="propertyGroupLayout">
         <property name="spacing">
          <number>12</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QComboBox" name="aiModelSelect">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="versionSelect">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="promptPartLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="promptLabel">
       <property name="text">
        <string>Prompt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="promptEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>243</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>243</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>Enter prompt here</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="triggerPartLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="triggerLabel">
       <property name="text">
        <string>Trigger</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="triggerTypeSelect">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>688</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="periodicPropertyContainer" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>8</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>46</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>46</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Every</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="periodicValueEdit">
        <property name="minimumSize">
         <size>
          <width>313</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>313</width>
          <height>40</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Enter a periodic value here</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="periodicUnitSelect">
        <property name="minimumSize">
         <size>
          <width>313</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>313</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../../Resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
